### PR TITLE
`contract-compile` and `contract-prove`

### PR DESCRIPTION
**Pull request by ChristianoBraga**
- Add shell scripts `contract-compile` and `contract-prove` to KPlutus executable directory.

  * Change `Makefile` to do so.
  * Add the above scripts.
  * Update `README.md` with a draft explanation on how to use the scripts.

- With this PR and #381, assuming the directory structure and files exlpained in `README.md/Compiling contracts`, one just need to hit `contract-contract` and `contract-prove` to get one's properties proven.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -346,7 +346,7 @@ kplutus-pyk:
 # Installing
 # ----------
 
-install_bins := kplc kplutus-covr
+install_bins := kplc kplutus-covr contract-compile contract-prove
```
Comment (gtrepta) [inline Makefile:349]: Could we make these into kplc commands instead? ie. `kplc contract-compile` and `kplc contract-prove`, so as not to put executables on the user's PATH that look irrelevant.

Comment (ChristianoBraga) [reply Makefile]: Makes sense, even though I think the commands are relevant.

Comment (gtrepta) [reply Makefile]: The names of the executables don't reference KPlutus in any way, that was my concern. In either case, I think adding CLI functionality means adding functionality to `kplc`.
